src/style.css

```diff
@@ -0,0 +1,89 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;

```

Comment:
redundant default property

Reply:
In the checklist(7-th paragraph) it is indicated that this is a good code.